lib/tapioca/dsl/compilers/active_record_relations.rb

```diff
@@ -813,6 +816,12 @@ def create_relation_method(
             return_type: association_return_type,
           )
         end
+
+        sig { params(constant: Module).returns(T::Boolean) }
+        def weak_find_each?(constant)
+          Object.const_defined?(:TapiocaWeakFindEach) &&
```

Comment:
Let's ensure this is documented. I don't see any current place for `StrongTypeGeneration` so I'd suggest a new section in the README, e.g. `Configuration Constants`.

Reply:
I don't think we should be using this approach for this. Let's make sure we can have real configuration without having to define modules in application and include them in our classes.